src/Foundation/Connect/code/Builders/Search/SearchQueryBuilder.cs

```diff
@@ -54,7 +54,7 @@ public IQueryable<CommerceSellableItemSearchResultItem> BuildProductsQuery(IQuer
             if (!string.IsNullOrWhiteSpace(searchKeyword))
             {
                 queryable = queryable.Where(
-                    item => item.Name.Contains(searchKeyword) || item.DisplayName.Contains(searchKeyword));
+                    item => item.Name.Contains(searchKeyword) || item["_displayname"].Contains(searchKeyword));
```

Comment:
Why we change to getting by field name?

Reply:
It wasn't working with `.DisplayName`. Looks like there are 2 of them. So I have to left it as it was before.